internal/configuration/config.go

```diff
@@ -44,6 +46,17 @@ type Config struct {
 	Daemon bool `yaml:"-"`
 }
 
+type EventsConfig struct {
+	Enabled bool                   `yaml:"enabled"`
+	Sinks   []*WavefrontSinkConfig `yaml:"sinks"`
+	Filters EventsFilter           `yaml:"filters"`
```

Comment:
We should support global filtering on the sink level too. Can add an `eventFilters` property on the sink for that similar to how we have for metrics.

But would it be possible to also support some pre-filtering for the events here, a sort of only collect events that are of certain types or such?

Reply:
we have just one events source with its own filters, adding event filtering on the skin will duplicate functionality.